app/assets/stylesheets/components/claim_deadline.scss

```diff
@@ -0,0 +1,25 @@
+@import "govuk-frontend-rails";
+
+%claim-deadline {
+  text-transform: lowercase;
+  letter-spacing: 0;
+  min-width: 4em;
+  text-align: center;
+}
```

Comment:
Is this a generic style for tags (or whatever we're calling those things)? Does it make sense to make it a generic component that we import?